src/core/opamStd.ml

```diff
@@ -554,8 +554,34 @@ end
 
 module OpamSys = struct
 
+  let is_windows = Sys.os_type = "Win32"
```

Comment:
is_windows == Sys.win32

Reply:
Yes. This code was there before OCaml 4.01.0, which introduced it, though.